assembly/src/lib.rs

```diff
@@ -59,6 +59,10 @@ const MAX_EXP_BITS: u8 = 64;
 /// The maximum length of a procedure's name.
 const MAX_PROC_NAME_LEN: u8 = 100;
 
+/// The required length of the hexadecimal representation for an input value when more than one hex
+/// input is provided to `push` without period separators.
```

Comment:
Can we give more context on this comment here? i.e. what push are we referring to?